### ISSUE TITLE
[feature request]  panorama video support for video call

### ISSUE THREAD

**Issue by wjhtinger** [vector-im/riot-web#3862]
In video call, can display the panorama video of the peer. 
I think the A-Frame can implement this goal. Or give some ideals, I try to do it myself.

**Comment by lampholder** [comment]
Are you suggesting something to take advantage of a 360 degree camera on the peer's computer?

**Comment by wjhtinger** [comment]
Yes. Just I want to implement VR video call by use of 360 degree camera or big wide angle(>180) fisheye camera.
As I know, A-frame can display the local camera vr stream（see https://github.com/jesstelford/aframe-video-billboard). But I am not sure if can display the live stream come from network.

**Comment by lampholder** [comment]
This is an interesting idea. I'd think it might be a little tricky to build in alongside the rest of the voip stuff robustly, but it's certainly something interesting to experiment with.

**Comment by wjhtinger** [comment]
add more project reference: https://matrix.org/vrdemo/#/room/#vrvc_young-thunderbird:conf.matrix.org
